README.rst

```diff
@@ -26,6 +26,8 @@ Next, both ``docker-compose.yml`` and ``docker-compose.dev.yml`` are required, s
 
   npm run dev
 
+NOTE: This might take signsfive_api to restart couple of time due to race condition with database starting up.
```

Comment:
https://hackernoon.com/preventing-race-conditions-in-docker-781854121ed3

Reply:
deafchi/signsfive-api/issues/26 will try to resolve this race conditions